x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/license/20_put_license.yml

```diff
@@ -127,6 +127,12 @@ teardown:
 ---
 "Current license is trial means not eligle to start trial":
 
+  # Check the current state of the license. If this is not a trail, then the rest of the test will fail (but be hard to diagnose)
```

Comment:
I think I know what this comment tries to say. But the words are not clear to me. Is it something like the following?
```suggestion
  # Check the current state of the license. If this is not a trail, then the rest of the test will fail (but the failure is hard to diagnose without this assertion)
```